spec/factories/users.rb

```diff
@@ -105,7 +105,7 @@
     end
 
     trait :loa3 do
-      authn_context { 'http://idmanagement.gov/ns/assurance/loa/3/vets' }
+      authn_context { 'http://idmanagement.gov/ns/assurance/loa/3' }
```

Comment:
```suggestion
      authn_context { LOA::IDME_LOA3 }
```

This trait has historically been for the `/vets` variety of an loa3 user. I'm surprised that this didn't break any existing specs... I guess we weren't using it anywhere?